### DISCUSSION TITLE
支持插件

### DISCUSSION THREAD

**Discussion by Skying-zhishi** [Meloong-Git/PCL#940] [提问] [answered]
好多人的奇思妙想基本上都进不去PCL2,加入插件的话不仅能扩展PCL2的功能，说不定能减少龙猫的更新工作量（雾

**Comment by WorldHim** [comment]
呃呃，但是你要想到做一个插件接口也很麻烦。
~~而且这种新功能提案能不能发issue啊啊啊啊啊啊啊啊啊~~

**Comment by Pigeon0v0** [comment, marked as answer]
#441
